shared-kcp/clusterrole.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
Maybe you could split the manifests from the scripts and put them in a `manifests` folder?

Reply:
Done.